mapbox_gl_web/lib/src/convert.dart

```diff
@@ -41,6 +38,10 @@ class Convert {
     if (options.containsKey('zoomGesturesEnabled')) {
       sink.setZoomGesturesEnabled(options['zoomGesturesEnabled']);
     }
+    if (options.containsKey('rotateGesturesEnabled')) {
+      //Should not be invoked before sink.setZoomGesturesEnabled()
+      sink.setRotateGesturesEnabled(options['rotateGesturesEnabled']);
```

Comment:
Could `setZoomGesturesEnabled` be affected by this modification?

Reply:
from what I can tell, it will impact all gestures which may use `keyboard` : 

`setZoomGesturesEnabled` controls: 

      _map.doubleClickZoom.enable();
      _map.boxZoom.enable();
      _map.scrollZoom.enable();
      _map.touchZoomRotate.enable();
      _map.keyboard.enable();  

and then `setRotateGesturesEnabled` controls:

      _map.dragRotate.enable();
      _map.touchZoomRotate.enableRotation();
      _map.keyboard.enable();

The KeyboardHandler in the current implementation seems to not offer a granular option to disable/enable certain keys or combinations of keys. 

I think a quick action is to simply call the methods in the `Convert` in the order of their likelihood: 

How often do you need a map that does not zoom but which you can rotate vs. A map which can be zoomed but you cannot rotate

Reply:
just to reiterate - with the current version one can:

disable zoom
disable zoom and rotation
but cannot only disable rotation

Is this correct?

Reply:
@felix-ht yes, this is correct.